### PR TITLE
Planner,MPP: check whether the order-by items can be pushed down to tiflash/tikv or not

### PR DESCRIPTION
**Pull request by fzhedu**
### What problem does this PR solve?

Issue Number: close #xxx
related to https://github.com/pingcap/tics/issues/3342

Problem Summary: agg functions do not check whether the order-by items can be pushed down to tiflash or not

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: planner/core/task.go [hunk 1]
```diff
@@ -1458,6 +1458,18 @@ func CheckAggCanPushCop(sctx sessionctx.Context, aggFuncs []*aggregation.AggFunc
 			ret = false
 			break
 		}
+		orderBySize := len(aggFunc.OrderByItems)
+		if orderBySize > 0 {
+			exprs := make([]expression.Expression, 0, orderBySize)
+			for _, item := range aggFunc.OrderByItems {
+				exprs = append(exprs, item.Expr)
+			}
+			if !expression.CanExprsPushDownWithExtraInfo(sc, exprs, client, storeType, aggFunc.Name == ast.AggFuncSum) {
```
Comment (windtalker) [inline planner/core/task.go:1467]: I think should pass `false` for the last parameter is ok.

**Comment by windtalker** [comment]
The title is not acurrate enough, this check is not only for TiFlash.

**Review by windtalker — APPROVED** [review]
LGTM

**Review by LittleFall — COMMENTED** [review]
would you mind adding a test to check the warning message, for example, in `TestShowWarningsForExprPushdown`?

**Review by LittleFall — APPROVED** [review]

**Pull request merged**